tensorflow/compiler/xla/stream_executor/rocm/rocm_helpers.cu.cc

```diff
@@ -74,5 +75,78 @@ void rocm_MakeBatchPointers(void* stream, char* base, int stride, int n,
       ptrs_out);
 }
 
```

Comment:
There seem to be quite a lot of long lines in this file. Can you please reformat to lines of length <= 80?